kalbarri-workshop.md

```diff
@@ -205,12 +205,14 @@ Boolean data is used to store true or false values. This is useful for storing t
 Boolean operations are useful for controlling the flow of programs (e.g. if X is true then do Y) and for filtering datasets based on a condition (e.g. get all cities with a population greater than 500,000). Let's experiment with boolean conditions. 
 
 ```js
-var demoBool = z == 4;
-print(demoBool);
+var demoBool = z;
```

Comment:
Consider splitting variable creation and logic over multiple lines for newcomers